### DISCUSSION TITLE
SQL Database Projects extension (0.21.0) issue.  Is this the right repo to add issues?

### DISCUSSION THREAD

**Discussion by piotr-labuda** [microsoft/azuredatastudio#21991] [Q&A] [answered]
Hi,
I haven't seen anywhere else mention this problem.  Is this the right repo to add issues about the extensions?
I have an existing database project and I wanted to test how the Database Project Extension will handle it.

Project is in repo. 
If looked from the repo page, the username file looks ok (user@company.com)
If I connect to the same repo from the Database Project Extension, the user files are displayed wrong and the SQL compare can't find the file (user%40company.com). so it's having issues with the @ character in the file name. 

If this is the right place I will create an issue with a bit more details.

Thanks,
N

**Comment by Charles-Gagnon** [comment, marked as answer]
For this extension, yes. In general if you have an issue with an extension you can generally find a link to the repo on the Extension Gallery page. Many extensions currently live in this repo, but there are a number of them which are external and the link there should bring you to the repo (if a public one exists)

![image](https://user-images.githubusercontent.com/28519865/220411932-80bbb2eb-fc29-46db-bf3e-9629cc9eb119.png)

So yes, please file an issue and we can take a look!
